samples/accountLinking/skill/variables.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 exports.user = function user(model) {
-  if (model.user) {
-    return model.user.email;
+  if (model.email) {
+    return model.email;
```

Comment:
is this correct?  seems like https://github.com/mediarain/voxa/pull/21/files#diff-b0d12e95adc3ef86d1cf02fa31ad2e56R25 would set the email in the `model.user.email` key